Docker/cpu/cpu.Dockerfile

```
@@ -30,7 +30,7 @@ RUN python -m pip install --upgrade pip \
     starlette-context==0.5.1
 
 # Install llama-cpp-python (build with OpenBLAS)
-RUN CMAKE_ARGS="-DGGML_BLAS=ON -DGGML_BLAS_VENDOR=OpenBLAS" pip install llama_cpp_python==0.3.19 --verbose
+RUN CMAKE_ARGS="-DGGML_BLAS=ON -DGGML_BLAS_VENDOR=OpenBLAS" pip install "llama-cpp-python>=0.3.19,<0.4" --verbose
```

Hadolint (2.14.0)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)